pkg/db-repo/model.go

```diff
@@ -24,6 +24,11 @@ type TimeStampable interface {
 	SetCreatedAt(createdAt *time.Time)
 }
 
+type HasTimestamps interface {
```

Comment:
usually we use the `*Aware` pattern like `TimestampAware`..

Reply:
Fixed.